packages/@aws-cdk/aws-eks/lib/k8s-cluster-autoscaler.ts

```diff
@@ -0,0 +1,389 @@
+import autoscaling = require('@aws-cdk/aws-autoscaling');
+import iam = require('@aws-cdk/aws-iam');
+import core = require('@aws-cdk/core');
+import cdk = require('@aws-cdk/core');
+import { Cluster } from './cluster';
+import { KubernetesResource } from './k8s-resource';
+
+/**
+ * A Cluster Autoscaler
+ */
+export interface IKubernetesClusterAutoscaler extends core.IConstruct {
```

Comment:
What's the motivation for implementing this as an interface? Do you anticipate people wanting to "import" an existing instance (which is our main motivation for the interface types across the library).

Reply:
Probably not, good call.